src/AsyncApiValidator.js

```diff
@@ -23,34 +22,20 @@ class AsyncApiValidator {
    */
   validate(key, payload) {
     if (!this._messages[key]) {
-      console.log(`key ${key} not found`)
+      console.error(`key ${key} not found`)
```

Comment:
Let's replace this logs with [debug](http://npmjs.com/package/debug) module? Or if you think it's required let's receive a custom log in the `options` parameter.

Reply:
will add this to TODO list. 👍